src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -384,11 +385,19 @@ export class HuiDialogEditCard
       return;
     }
     this._saving = true;
-    await this._params!.saveCardConfig(this._cardConfig!);
-    this._saving = false;
-    this._dirty = false;
-    showSaveSuccessToast(this, this.hass);
-    this.closeDialog();
+    try {
+      await this._params!.saveCardConfig(this._cardConfig!);
+      this._saving = false;
+      this._dirty = false;
+      showSaveSuccessToast(this, this.hass);
+      this.closeDialog();
+    } catch (err: any) {
+      showToast(this, {
+        message: err.message,
+        duration: 5,
+      });
```

Comment:
Should it be a toast? Normally we use a `ha-alert` inside the dialog?

Reply:
Yeah, but I don't think we have space for it and I didn't wanted to make too many change in this PR. Also, it should never happens, right?